src/server/static/js/lines.js

```diff
@@ -299,7 +299,7 @@ function AddCriticalCareTrace(traces) {
   traces.push({
     x: d3.extent(traces[0].x),
     y: [capacity, capacity],
-    name: "Hospital critical care capacity",
+    name: "All ICU beds are needed for critical cases",
```

Comment:
I don't think this captures what is meant. Maybe something more like `"Hospital capacity for COVID-19"`. I vote that we make this the actual meaning of the line and change how we calculate the number accordingly.

Otherwise, there's no intuitive way to explain what this number is. It isn't total critical care capacity because it's actually about 20x that because it's been adjusted to match the infection line, and it isn't COVID-19 capacity because it uses _all_ critical beds, not just those available.